tests/jquery.flot.touch.Test.js

```diff
@@ -401,4 +401,67 @@ describe("flot touch plugin", function () {
             expect(spy).not.toHaveBeenCalled();
         });
     });
+
+    describe('tap', function() {
```

Comment:
I wish to see a test which take into account the pan interaction as well. Basically, are the pan events expected to happen while tapping? Either way, this should be 'documented' with a test.